app/src/app/core/services/elasticsearch/data.service.ts

```diff
@@ -24,7 +25,8 @@ export class DataService {
   constructor(private http: HttpClient, @Inject(LOCALE_ID) localeId: string) {
     // build backend api url with specific index by localeId
     this.ISO_639_1_LOCALE = localeId.substr(0, 2);
-    this.baseUrl = elasticEnvironment.serverURI + '/' + (this.ISO_639_1_LOCALE || 'en') + '/_search';
+    this.searchEndPoint = environment.elastic.base + '/' + (this.ISO_639_1_LOCALE || 'en') + '/_search';
+    this.countEndPoint = environment.elastic.base + '/' + (this.ISO_639_1_LOCALE || 'en') + '/_count';
```

Comment:
I would keep the `baseUrl` and just append for the endpoints:
```suggestion
    this.baseUrl = environment.elastic.base + '/' + (this.ISO_639_1_LOCALE || 'en')';
    this.searchEndPoint = this.baseUrl + '/_search';
    this.countEndPoint = this.baseUrl + '/_count';
```

Reply:
Not sure if I understand your point correctly, but does it make a difference? It would just be a duplicate of environment.elastic.base I guess, so I think this is as good as having both endpoints.